aspnetcore/grpc/troubleshoot.md

```diff
@@ -101,43 +104,20 @@ To work around this issue, configure Kestrel and the gRPC client to use HTTP/2 *
 
 Kestrel must configure an HTTP/2 endpoint without TLS in `Program.cs`:
 
-:::moniker range=">= aspnetcore-6.0"
 ```csharp
 var builder = WebApplication.CreateBuilder(args);
 
 builder.WebHost.ConfigureKestrel(options =>
 {
     // Setup a HTTP/2 endpoint without TLS.
-    options.ListenLocalhost(5000, o => o.Protocols =
+    options.ListenLocalhost(<5287>, o => o.Protocols =
         HttpProtocols.Http2);
 });
 ```
-:::moniker-end
 
-:::moniker range="< aspnetcore-6.0"
-```csharp
-public static IHostBuilder CreateHostBuilder(string[] args) =>
-    Host.CreateDefaultBuilder(args)
-        .ConfigureWebHostDefaults(webBuilder =>
-        {
-            webBuilder.ConfigureKestrel(options =>
-            {
-                // Setup a HTTP/2 endpoint without TLS.
-                options.ListenLocalhost(5000, o => o.Protocols = 
-                    HttpProtocols.Http2);
-            });
-            webBuilder.UseStartup<Startup>();
-        });
-```
-:::moniker-end
+* In the preceding highlighted code, replace the localhost port number `5287` with the `HTTP` (not `HTTPS`) port number specified in `Properties/launchSettings.json` within the gRPC service project.
```

Comment:
@JamesNK, this is what was added, a clarification on the port number for 6.0 which would be randomly assigned at project creation.   Looking over comments on this topic and on forums, it looks like folks are occasionally either specifying the https port or leaving it as `5000` by mistake.